server/trillian_log_server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang
+
+ENV DB_USER=test \
```

Comment:
ENV allows the arguments to be overridden at runtime which might be even more flexible than fixing them at build time.